.github/workflows/tests.yaml

```diff
@@ -22,18 +22,11 @@ jobs:
                     coverage: none
 
             # Use library versions compatible with PHP 7.1
-            -   run: |
-                    mkdir temp && cd temp
-                    composer require "doctrine/orm:~2.7.5" --ansi --no-interaction
-                    composer require "phpunit/phpunit:~7.5" --dev --ansi --no-interaction
+            # -   run: composer require "doctrine/orm:~2.7.5" --ansi --no-interaction
 
-            # Apply Rector on a few sources
-            -   run: |
-                    cd temp
-                    ../bin/rector process ../ci/downgrade/tests/Finalize/Fixture/Source --config ../ci/downgrade/rector-test-downgrade.php --ansi
+            -   run: wget -O phpunit https://phar.phpunit.de/phpunit-7.phar
+            -   run: chmod +x phpunit
 
-            # Validate the results are the expected ones
-            -   run: |
-                    cd temp
-                    vendor/bin/phpunit ../ci/downgrade/tests/
+#            -   run: composer require "phpunit/phpunit:~7.5" --dev --ansi --no-interaction
 
+            -   run: php phpunit
```

Comment:
cc @samsonasik

Reply:
@TomasVotruba after composer update, it got error:

```bash
Class 'Rector\Core\Stubs\StubLoader' not found
```

![Screen Shot 2021-02-10 at 16 03 53](https://user-images.githubusercontent.com/459648/107488469-fba5c900-6bb9-11eb-91ef-85a8c47e6a88.png)

The `Stubs` directory seems excluded in possibly the following script:

https://github.com/rectorphp/rector/blob/69d5980d1dcaff72db56d93ded07912d503620fd/.github/workflows/build_php71.yaml#L31

by `-x` option for `stubs** `